src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
@heejaechang You explained this to me one time, but i forget the reason now.  Why don't we just use the actual Workspace Document order when computing checksums?  Thanks!